### DISCUSSION TITLE
more interesting enemies

### DISCUSSION THREAD

**Discussion by Bajad3rka** [Quit975/DuckyGame#48] [General]
imo green enemies should be pink or pastel purple, it will be more cool really! And what do u think about angry faces of enemies? Or maybe enemies should be an animal which is dangerous for ducky like fox or wolf or idk eagle

**Comment by Daria13szamman** [comment]
Sounds like a cute idea :D. I think fox enemy would be fun to make . Here we have more predators that attack ducks https://deltawaterfowl.org/top-duck-craving-predators/

**Comment by Quit975** [comment]
Very similar discussion to https://github.com/Quit975/DuckyGame/discussions/26

Let's not open discussions "we need more diverse enemies". Instead, open a discussion "Idea for enemy: xyz" :)
